R/r.R

```diff
@@ -158,6 +158,30 @@ r_cmd_install <- function(package, path, ...) {
     defer(unlink(path, recursive = TRUE))
   }
 
+  # on Windows, just unpack binary archives directly into the library tree
+  # https://github.com/rstudio/renv/issues/1359
+  unpack <-
+    renv_file_type(path) == "file" &&
+    renv_archive_type(path) %in% "zip" &&
+    renv_package_type(path) %in% "binary"
+
+  if (unpack) {
+
+    # unpack into temporary directory in library path
+    library <- renv_libpaths_active()
+    exdir <- renv_scope_tempfile(package, tmpdir = library)
+    unzip(path, exdir = exdir)
```

Comment:
It just occurred to me, but can we just install from that directory?

Reply:
You mean unpack into the library path directly? We could, but this is a bit safer in case multiple processes are trying to install into the same library at the same time. (really, we should be using locking but I got lazy).

Reply:
No, I mean call `install.packages()` (or the `R CMD install` equivalent) on the unzipped directory, instead of the zipped file.

Reply:
Ohh, yes! I'll try that.